packages/blockchain-wallet-v4/src/redux/payment/nfts/utils.ts

```diff
@@ -1150,14 +1174,15 @@ async function _initializeProxy(signer): Promise<string> {
     proxyRegistry_ABI,
     signer
   )
-  const gasPrice = await signer.getGasPrice()
   const txnData = {
     from: accountAddress,
-    gasLimit: 410_000,
-    gasPrice: parseInt(gasPrice._hex)
+    gasLimit: 410_000
   }
-  const gasLimitEstimated = await wyvernProxyRegistry.estimateGas.registerProxy(txnData)
-  txnData.gasLimit = parseInt(gasLimitEstimated._hex)
+  txnData.gasLimit = await safeGasEstimation(
```

Comment:
should wrap calls to `safeGasEstimation` in a try catch since it can throw?

Reply:
What I need to do to spend it